source/agora/crypto/Schnorr.d

```diff
@@ -169,6 +163,7 @@ public Signature sign (T) (in Scalar privateKey, scope const auto ref T data)
 
 /// Sign with a given `r` (warning: `r` should never be reused with `x`)
 public Signature sign (T) (in Pair kp, in Pair r, scope const auto ref T data)
+    nothrow @nogc @safe
```

Comment:
This looks unrelated (and unnecessary?)

Reply:
Reverted. I will add another commit to remove the other overloaded functions which also have unnecessary attributes.